core/dom/text.js

```diff
@@ -76,6 +76,17 @@ CKEDITOR.tools.extend( CKEDITOR.dom.text.prototype, {
 		this.$.nodeValue = text;
 	},
 
+	/**
+	 * Checks whether a node is empty or is
+	 * {@link CKEDITOR.dom.selection#FILLING_CHAR_SEQUENCE FILLING_CHAR_SEQUENCE} string.
+	 *
+	 * @since 4.13.0
+	 * @returns {Boolean}
+	 */
+	isEmpty: function() {
```

Comment:
I'd extend this a little, as described in https://github.com/ckeditor/ckeditor-dev/issues/3326#issuecomment-520221901.